src/python/library/requirements/requirements_grpc.txt

```diff
@@ -25,6 +25,8 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 protobuf>=3.5.0
-grpcio>=1.31.0
+# There is memory leak in later Python GRPC (1.43.0 to be specific),
```

Comment:
Does gRPC have an open github issue tracking the memory leak? If yes, then it would be helpful to include the link in this comment.